docs/guides/deploy-guide/manager.md

```diff
@@ -112,13 +116,15 @@ in the Ansible documentation.
 | `ANSIBLE_BECOME_ASK_PASS` | Boolean | Toggle to prompt for privilege escalation password.                                                                                                                           |
 | `ANSIBLE_SSH_ARGS`        | String  | If set, this will override the Ansible default ssh arguments.                                                                                                                 |
 | `ANSIBLE_USER`            | String  | The user Ansible ‘logs in’ as.                                                                                                                                                |
-To verify the creation of the operator user, use the private key file `id_rsa.operator`. Make
+
+
+To verify the proper creation of the operator user, use the private key file `id_rsa.operator`. Make
 sure you purge all keys from ssh-agent identity cache using `ssh-add -D`. You can print the list
 using `ssh-add -l`. The list should be empty.
 
 ```
 ssh-add -D
-ssh -o IdentitiesOnly=yes -i id_rsa.operator dragon@testbed-manager
+ssh -o IdentitiesOnly=yes -i  PATH_TO_YOUR_SECRETS/id_rsa.operator dragon@YOUR_MANAGER_NODE
```

Comment:
PATH_TO_YOUR_SECRETS is not required. The id_rsa.operator file is available at this location when running the manager seed steps.

Reply:
The user is probably located in the "<base-dir>/environments/manager" directory at this point. I wanted to make it clearer with this adjustment that the file is not located in the directory. On closer inspection, however, this is not true, because it is already the case on the seed node.
Removed "PATH_TO_YOUR_SECRETS/" now.

Reply:
The id_rsa.operator file is available in environments/manager when seeding the manager.